recipes/napari-microscope/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "napari-microscope" %}
+{% set version = "0.0.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/napari-microscope-{{ version }}.tar.gz
+  sha256: 65efe52ed6ac1ee312256d87ec8e3945738cd93449f0a55c6bf75f48cc232fe1
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - microscope
+    - napari-plugin-engine
+    - pyro4
+    - python >=3.6
+
+test:
+  imports:
+    - napari_microscope
+  commands:
+    - pip list
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://pypi.org/project/napari-microscope/
+  summary: Napari plugin for Microscope.
+  license: GPL-3.0-or-later
+  license_file: COPYING
```

Comment:
```suggestion
  license_file: COPYING
  dev_url: https://github.com/carandraug/napari-microscope
```

Reply:
Oops, I didn't commit this change. Tag me when the feedstock gets created @goanpeca, please!